includes/Renderer/Story/HTML.php

```diff
@@ -128,11 +139,33 @@ public function render() {
 	 * @return array Sanitizers.
 	 */
 	public function add_web_stories_amp_content_sanitizers( $sanitizers ) {
+		$video_cache_enabled = $this->experiments->is_experiment_enabled( 'videoCache' ) && (bool) get_option( Settings::SETTING_NAME_VIDEO_CACHE );
+
 		$sanitizers[ Story_Sanitizer::class ] = [
 			'publisher_logo'             => $this->get_publisher_logo(),
 			'publisher'                  => $this->get_publisher_name(),
 			'publisher_logo_placeholder' => $this->get_publisher_logo_placeholder(),
 			'poster_images'              => $this->get_poster_images(),
+			'video_cache'                => $video_cache_enabled,
+		];
+
+		return $sanitizers;
+	}
+
+	/**
+	 * Filters the AMP plugin's sanitizers.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param array $sanitizers Sanitizers.
+	 * @return array Sanitizers.
+	 */
+	public function add_amp_content_sanitizers( $sanitizers ) {
+		$sanitizers[ AMP_Story_Sanitizer::class ] = [
+			'publisher_logo'             => $this->get_publisher_logo(),
+			'publisher'                  => $this->get_publisher_name(),
+			'publisher_logo_placeholder' => $this->get_publisher_logo_placeholder(),
+			'poster_images'              => $this->get_poster_images(),
```

Comment:
Can you explain this change?

Reply:
Whoops, added by accident. Need to remove it.